AEA-1242 Update Q&A section and ledger section in docs (#2193)
fetchai/agents-aea: docs/index.md

<iframe width="560" height="315" src="https://www.youtube.com/embed/xpJA4IT5X88" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>

> AEAs act independently of constant input from their owner and autonomously execute actions to achieve their prescribed goal. Their goal is to create economic value for you, their owner, in a clearly defined domain. AEAs have a wide range of <a href="app-areas">application areas</a> and we provide <a href="demos">demo guides</a> to highlight some examples.

Reviewer: ```suggestion
AEAs act independently of constant input from their owner and autonomously execute actions to achieve their prescribed goals. Their purpose is to create economic value for you, their owner, in clearly defined domains. AEAs have a wide range of <a href="app-areas">application areas</a> and we provide <a href="demos">demo guides</a> to highlight examples of their use cases.
```

<!-- Autonomous Economic Agents are digital entities that run complex dynamic decision-making algorithms for application owners and clients. -->

AEAs are not:

> * just any <a href="https://en.wikipedia.org/wiki/Software_agent"  target="_blank">agents</a>: AEAs' purpose is to generate economic value in a multi-stakeholder environment with competing incentives between agents. They represent humans, organisations or things.

Reviewer: ```suggestion
* just any <a href="https://en.wikipedia.org/wiki/Software_agent"  target="_blank">agents</a>: AEAs' purpose is to generate economic value in a multi-stakeholder environment with competing incentives between agents. They represent humans, organisations or objects.
```

* APIs or sensors which do not have agency.
* <a href="https://en.wikipedia.org/wiki/Smart_contract" target="_blank">smart contracts</a> which do not display any proactiveness and are purely reactive to external requests (=contract calls and transactions). 

> * <a href="https://en.wikipedia.org/wiki/Artificial_general_intelligence" target="_blank">artificial general intelligence (AGI)</a>: AEAs can have a very narrow goal directed focus involving some economic gain and implemented via simple conditional logic.

Reviewer: ```suggestion
* <a href="https://en.wikipedia.org/wiki/Artificial_general_intelligence" target="_blank">artificial general intelligence (AGI)</a>: AEAs can have a very narrow, goal directed focus involving some economic gain and can be implemented via simple conditional logic.
```

## What is the AEA Framework?

The AEA framework is a Python-based development suite which equips you with an efficient and accessible set of tools for building AEAs. The framework is modular, extensible, and composable. It attempts to make agent development as straightforward an experience as possible, similar to web development using popular web frameworks.

> AEAs achieve their goals with the help of a decentralized search & discovery system for agents the <a href="oef-ledger">simple Open Economic Framework (sOEF)</a>, the <a href="acn">Agent Communication Network</a> - a decentralized communication system for agents - and using <a href="oef-ledger">Fetch.ai's blockchain</a> as a financial settlement and commitment layer. Third-party blockchains, such as <a href="https://ethereum.org/en/" target="_blank">Ethereum</a>, may also allow AEA integration.

Reviewer: ```suggestion
AEAs achieve their goals with the help of a decentralized search & discovery system for agents the <a href="oef-ledger">simple Open Economic Framework (sOEF)</a>, the <a href="acn">Agent Communication Network</a> - a decentralized communication system for agents - and using <a href="oef-ledger">Fetch.ai's blockchain</a> as a financial settlement and commitment layer. AEA could also be integrated with third-party blockchains, such as <a href="https://ethereum.org/en/" target="_blank">Ethereum</a>.
```

Reviewer: The `decentralized` adj. for search and discovery may be problematic when a link to sOEF follows it because the latter isn't decentralised.
